### PR TITLE
Updates to code processing

### PR DESCRIPTION
**Pull request by neubig**
This PR makes several updates to code processing:

* Removes safety filters from gemini
* Changes the prompt to one requested by the Gemini team, which results in 1-2 point increase across all models
* Passes in max_tokens to the completion call and sets it to 512
* Sets the temperature to 0.0 and top_p to 1.0
* Passes in vertex_project and vertex_location through the command line
* Strips extra English language comments from after the implemented function through a regex, as this was a major failure case that's easy to fix through post-processing
* Upgrades the deprecated Hugging Face evaluate calls

### CODE REVIEW

File: benchmarking/Code/run_code.py [hunk 1]
```diff
@@ -3,31 +3,66 @@
...
             },
-            {"role": "user", "content": prompt},
-        ],
+        ]
+        extra_kwargs = {
+            "safety_settings": safety_settings,
+        }
+    elif "mixtral" in model:
+        extra_kwargs = {
+            "stop": ["###"],
```
Comment (Sparkier) [inline benchmarking/Code/run_code.py:50]: Not needed for Gemini?

**Review by Sparkier — APPROVED** [review]
LGTM but I have no experience with the litellm calls.

File: benchmarking/Code/run_code.py (hunk 1 above)
Comment (Sparkier) [reply benchmarking/Code/run_code.py:50]: Stop param I'm talking about.

Comment (neubig) [reply benchmarking/Code/run_code.py:50]: Not supported by gemini as far as I can tell (also not needed).

**Pull request merged**